### PR TITLE
POM > AT_16.04 _001| Verify that the Multibranch Pipeline is moved to an existing folder using dropdown

### PR DESCRIPTION
**Pull request by YurySelivanov**
https://trello.com/c/rF114Wpy/1719-rf-pom-at1604-001-verify-that-the-multibranch-pipeline-is-moved-to-an-existing-folder-using-dropdown

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -103,12 +103,14 @@ Cypress.Commands.add('addFolderDescription', (folderDescription) => {
         .clickJenkinsHomeLink()
 });          
 
-Cypress.Commands.add('createMultiBranchPipeline', (name) => {
+Cypress.Commands.add('createMultBranchPipeline', (name) => {
     homePage
```
Comment (LanaBondarenko) [inline cypress/support/commands.js:107]: What was the purpose of the name change? As I can see this precondition is already being used in several files/tests.

**Review by SvetlanaMag — APPROVED** [review]

**Review by Forostinko — APPROVED** [review]

File: cypress/support/commands.js (hunk 1 above)
Comment (YurySelivanov) [reply cypress/support/commands.js:107]: don't produce dublicates of methods, I just have edited one of these

File: cypress/pageObjects/HomePage.js [hunk 2]
```diff
@@ -207,6 +211,23 @@ class HomePage {
...
+  hoverAndClickProjectDrpDwn(projectName) {
+    this.getProjectName(projectName).realHover();
+    this.getProjectDrpDwn(projectName).click();
+    return this;
+  }
+
+  clickProjectName(projectName) {
+    this.getProjectName(projectName).click();   
+    return new FolderPage;
+  }
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:229]: Such methods are already in the file.

Comment (YurySelivanov) [reply cypress/pageObjects/HomePage.js:229]: I know but if I would use existing methods I needed to use loop for enumeration and complex the code so I decided to create new methods with parameter

Comment (Rom-T) [reply cypress/pageObjects/HomePage.js:229]: How is this hoverAndClickProjectDrpDwnBtn(projectName) method different from yours hoverAndClickProjectDrpDwn(projectName). Or clickFolderNameLink from this clickProjectName

**Pull request merged**